template/src/assets/stylesheets/base-styles.scss

```diff
@@ -13,6 +13,7 @@
 
 body {
   margin: 0;
+  font-size: 16px;
```

Comment:
This will force the user to always have font size 16 right? I'm thinking that we should probably move to a rem-based font size and line height definition. Maybe not for this PR, but worth thinking about

Reply:
The reason is as the Figma specifies we will use 16px base, and then use rems everywhere else as needed. This is just to set the base font size for that.